catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/ResourceCacheImpl.java

```diff
@@ -109,16 +108,13 @@ private Config getHazelcastConfig(BundleContext context, String xmlConfigFilenam
         cfg = xmlConfigBuilder.build();
         LOGGER.debug(
             "Successfully built hazelcast config from XML config file {}", xmlConfigFilename);
-      } catch (FileNotFoundException e) {
+      } catch (IOException e) {
         LOGGER.info(
-            "FileNotFoundException trying to build hazelcast config from XML file "
+            e.getClass().getSimpleName()
```

Comment:
Shouldn't this still be parameterized?

Reply:
@bdeining I'll use a StringBuilder to build this message